<!DOCTYPE HTML>
<!--
	Read Only by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Nitya Thakkar</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<meta name="description" content="Nitya's Website">
    	<meta name="author" content="Nitya Thakkar">
		<meta name="google-site-verification" content="doL8XMtt4cn8S609D3Na_zp4sztpcI8apU0I3r5eZI4" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Header -->
			<section id="header">
				<header>

					<span class="image avatar"><img src="files/IMG_2055.jpg" alt="" /></span>
					<h1 id="logo"><a href="#">Nitya Thakkar</a></h1>
					<p>Computer Science at Brown University</p>

				</header>
				<nav id="nav">
					<ul>
						<li><a href="#about" class="active">About Me</a></li>
						<li><a href="#education">Education</a></li>
						<li><a href="#research">Research</a></li>
						<li><a href="#contact">Contact</a></li>
						<li><a href="files\Resume.pdf" target="_blank" class="button">Curriculum Vitae</a></li>
					</ul>
				</nav>
				<footer>
					<ul class="icons">
						<li><a href="https://www.linkedin.com/in/nitya-thakkar-a08a601a6" target="_blank" class="icon brands fa-linkedin-in"> <span class="label">LinkedIn</span></a></li>
						<li><a href="https://github.com/nityathakkar" target="_blank" class="icon brands fa-github"> <span class="label">Github</span></a></li>
						<li><a href="#contact" class="icon solid fa-envelope"><span class="label">nitya_thakkar -at- brown.edu</span></a></li>

						<!-- <li><a href="#" class="icon brands fa-linkedin-in"><span class="label">LinkedIn</span></a></li> -->
						<!-- <li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li> -->
						<!-- <li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li> -->
						<!-- <li><a href="#" class="icon brands fa-github"><span class="label">Github</span></a></li> -->
					</ul>
				</footer>
			</section>

			
		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">

						<!-- One -->
							<section id="about">
				
								<div class="container">
									<header class="major">
										<h2>Nitya Thakkar</h2>
										<p>Hi! I am a senior at Brown University studying Computer Science, with a specific interest in Machine Learning and Computational Biology.</p>
									</header>
								</div>
							</section>
							
						<!-- Two -->
							<section id="education">
								<div class="container">
									
										<h3>Education</h3>
										<div class="features">
											<article>

													<p><b>Brown University</b>, Sc.B. Computer Science, June 2023
													<br><i>Activities</i>: Meiklejohn Peer Advisor, Women in CS mentor, Brown Elementary Afterschool Mentoring volunteer, <i>The Brown Daily Herald</i> staff writer, Brown Abhinaya - classical Indian dance team (co-captain 2021-22), Full Stack at Brown software engineer
													<br><i>TA Experience</i>: Deep Learning, Fall 2022 and Spring 2022; Introduction to Computer Systems, Fall 2021; Linear Algebra, Spring 2021
													<br><b>St. Paul Academy and Summit School</b>, June 2019</p>
												
											</article>
											<article>

													<h4>Coursework at Brown</h4>
													<p><b>Computer Science</b>: Deep Learning, Machine Learning, Computational Molecular Biology, Data Science, Computer Vision, Introduction to OOP and Computer Science, Introduction to Algorithms and Data Structures, Computer Systems
													<br><b>Math</b>: Multivariable Calculus, Linear Algebra, Discrete Structures and Probability, Statistical Inference and Probability
													<br><b>Other</b>: Genetics, Biochemistry, Chemistry, Organic Chemistry</p>
												
											</article>
											<article>

													<h4>Programming Skills</h4>
													<p><b>Proficient</b>: Python, Java, C, PyTorch, and TensorFlow
													<br><b>Experience with</b>: HTML/CSS, JavaScript, React, R, x86-64 Assembly Language, LaTeX, and Git</p>

											</article>
											<article>

													<h4>Awards and Accomplishments</h4>
													<p><a href="https://cra.org/2022-outstanding-undergraduate-researcher-award-recipients/" target="_blank"><b>Computing Research Association's (CRA) Outstanding Undergraduate Researcher Award 2022 Honorable Mention </b></a>: This prestigious award recognizes undergraduate students who show outstanding research potential in an area of computing research.</p>
											</article>
										</div>

								</div>
							</section>

						<!-- Three -->
							<section id="research">
								<div class="container">
									<h3>Research Experience</h3>
									<div class="features">
										<article>

											<p> <a href="https://www.microsoft.com/en-us/research/theme/biomedical-ml/" target="_blank">Microsoft Research, Biomedical ML</a>
											<br><i>Dr. Kevin Yang, May 2022 - Present</i>
											<br>
											The goal of this project is to use denoising diffusion probabilistic models (diffusion models) to generate protein sequences. Specifically, our hypothesis is that Order Agnostic Autoregressive Diffusion Model in a two-dimensional space on Multiple Sequence Alignments (MSAs) will be able to capture the natural variation in the protein sequences and generate new proteins with desired structures and functions. The downstream applications of this project can have implications for protein engineering, since it may allow us to generate novel protein sequences with desired functionality (as protein structure dictates function); this can enable targeted protein therapies.
											<br> </p>
											
										</article>
										<article>

												<p> <a href="https://rsinghlab.org/author/ritambharasingh/" target="_blank">Brown University, Computational Biology Lab</a>
												<br><i>Dr. Ritambhara Singh, Jan. 2020 - Present</i>
												<br>

												<br> I am working on a project to predict gene expression values in glioblastomas, which is cancer in the brain that is 
												characterized by its aggressive nature. My aim is to predict these gene expression values from other epigenetic data to better understand the 
												genetic mechanisms in brain tumors. We found that a variety of baseline machine learning models, such as support vector 
												machines and random forest models did not learn many significant relationships. I am currently training a Convolutional Neural 
												Network (CNN) to accomplish this task, with the hopes that a more complex model will be able to learn the relationships more accurately.

												<br>
												
												<br> <a href="https://doi.org/10.1101/2022.04.19.488754" target="_blank">Predicting A/B compartments from histone modifications using deep learning</a> <br> Co-first author on a project to predict three-dimensional (3D) organization of our DNA from one-dimensional (1D) biological experiments 
												to understand gene interaction mechanisms. Understanding 3D spatial DNA organization is critical to understanding gene interactions; however, it is currently 
												difficult and expensive to manually obtain 3D data. I hence helped create a recurrent neural network (RNN) to predict the 3D DNA organization 
												from easily available 1D information about biological factors. This model outperformed the mean baseline model across all cell lines, which 
												indicates it is able to accurately predict the 3D interactions. I also implemented and ran baseline methods to compare the accuracy of my model, 
												and improved the data pre-processing pipeline to increase the resolution of the data, which allowed us to view regions of the genome more precisely. 
												Languages used: Python, R, Git</p>
												<a href="https://doi.org/10.1101/2022.04.19.488754" target="_blank" class="button">Link to Paper</a>
												
										</article>
										<article>

											<p><a href="https://www.broadinstitute.org/" target="_blank">Research Assistant at the Broad Institute of MIT and Harvard</a>
											<br><i>Dr. Neriman Tokcan, June 2021 - Dec. 2021</i>
											<br>
											<br> I was selected for the prestigious Broad Summer Research Program (2021), where I conducted research in Dr. Todd Golub’s and Dr. Caroline Uhler’s 
											labs under the mentorship of Neriman Tokcan, Ph.D. I worked with the lymphatic system cancer, Classical Hodgkin’s Lymphoma, 
											which is unique since it has a diverse composition of tumor cells and immune cells. Previous studies have found these tumor 
											cells are dependent on their environment for survival; however, not much is known about what specific cells and genes contribute 
											to their survival. Therefore, the goal of my project was to develop new computational methods to analyze spatial data that captures 
											the tumor cell environment. I implemented a feed-forward neural network to identify the types of cells in spatial data from gene expression data. 
											I also created a new method to analyze the spatial relationship and interactions between cells in the microenvironment. I anticipate these methods 
											will facilitate the downstream development of more targeted cancer therapies if wet-lab researchers target specific immune cells that support the 
											survival of cancerous cells. I presented this project at the Annual Biomedical Research Conference for Minority Students in November. Languages used: Python, R</p>
											<!-- <ul class="actions"> -->
											<a href="files\broad_poster.pdf" target="_blank" class="button">Poster PDF</a>
											<!-- </ul> -->

										</article>
										<article>

												<p> <a href="https://www.umass.edu/foodsci/faculty/yeonhwa-park" target="_blank">University of Massachusetts - Amherst, Food Science Lab</a>
												<br><i>Dr. Yeonhwa Park, July 2018 - Aug. 2018</i>
												<br>
												<br> The goal of my project was to determine the effects of Sulforaphane, a compound found in cruciferous vegetables, on aging, obesity and oxidative 
												stress in Caenorhabditis elegans. Previous studies have found it to have anticarcinogenic effects, and I sought to determine if it could have other 
												positive effects on health. I independently ran PCR analyses on specific genes, and conducted various wet-lab assays. The results indicated that 
												Sulforaphane may improve oxidative stress response in C. elegans through the post-translational regulation of SKN-1, which indicates that it may 
												slow aging. I received awards for both my poster presentation and paper at the Twin Cities Regional Science Fair and Minnesota State Science Fair. 
												I also qualified as a finalist for the prestigious Intel International Science and Engineering Fair, which I attended in May 2019. Through this 
												project, I made extensive use of different data analysis processes using Excel, Statistical Analysis Software (one-way ANOVA and chi-square test), 
												GraphPad Prism (Logrank test) and SPSS. </p>
													<!-- <ul class="actions"> -->
														<a href="files\Poster.pdf" target="_blank" class="button">Poster PDF</a>
													<!-- </ul> -->
													<!-- <ul class="actions"> -->
														<a href="files\Paper.pdf" target="_blank" class="button">Paper PDF</a>
													<!-- </ul> -->

										</article>

									</div>

								</div>
							</section>

						<!-- Four -->
							<section id="contact">
								<div class="container">
									<h3>Contact</h3>
										<section>
											<!-- <ul class="contact"> -->
												<!-- <li> -->
													<i class="fa fa-envelope fa-fw"></i> <span class="network-name">nitya_thakkar -at- brown.edu</span>
													<!-- <ul class="icons">
														<a href="https://www.linkedin.com/in/nitya-thakkar-a08a601a6" target="_blank" class="icon brands fa-linkedin-in"> <span class="label">LinkedIn</span></a>
													</ul>
													<ul class="icons">
														<a href="https://github.com/nityathakkar" target="_blank" class="icon brands fa-github"> <span class="label">Github</span></a>
													</ul>
													<ul class="actions">
														<li><a href="files\Resume.pdf" target="_blank" class="button">Curriculum Vitae</a></li>
													</ul>
												</li> -->
											<!-- </ul> -->
										</section>

								</div>
							</section>
					</div>

				<!-- Footer -->
					<section id="footer">
						<div class="container">
							<ul class="copyright">
								<li>&copy; Untitled. All rights reserved.</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
							</ul>
						</div>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>